examples/event_listener/README.md

```diff
@@ -0,0 +1,32 @@
+# Astarte Device SDK Python Example device
+This is an example usage of Device connection publication of values on either
+Datastream and Properties interfaces.
```

Comment:
Title and abstract do not reflect the purpose of the example. You could say something like:
> This is an example of how to use the Device SDK to connect a device to Astarte and handle incoming messages sent to the device by third parties.